setup_gmail.py

```diff
@@ -190,7 +191,7 @@ def get_service():
 
 class GmailPatchsetFetcher(object):
```

Comment:
I don't think this class is used anywhere. You could probably just remove it.

Reply:
Yeah, it's not used.
I was debating removing it, but deleting code felt a bit out-of-scope for these patches.

I'll submit as-is for now.